trade-finance-manager-api/api-tests/v1/utils/updateFacilityAmendment.js

```diff
@@ -0,0 +1,6 @@
+const app = require('../../../src/createApp');
```

Comment:
can we name the file `updateFacilityAmendment.util.js`?

Reply:
done